homeassistant/components/climacell/sensor.py

```diff
@@ -80,7 +80,7 @@ def _state(self) -> str | int | float | None:
         """Return the raw state."""
 
     @property
-    def state(self) -> str | int | float | None:
+    def native_value(self) -> str | int | float | None:
```

Comment:
Just to confirm, the `native_value` will get returned as the state if this is not a temperature device class entity with a temperature unit of measurement, right?

Reply:
Yeah, that's right